src/database/firebase.js

```diff
@@ -27,6 +27,48 @@ const userCollection = database.collection('users');
 const commentCollection = database.collection('comments');
 const additionalCollection = database.collection('additional');
 
+/**
+ * addTreeLocationTemp temporary function that adds a random location if the location is null
+ * imports Math and the map values from Default locations
```

Comment:
This line is unnecessary, it's already evident what this code uses

Reply:
we should keep this comment in order to more easily delete this function at deployment